docs/source/usage.rst

```diff
@@ -60,7 +60,7 @@ Positions:
   - ``T``: sending
   - ``t``: idle (shown after successful sending)
   - ``3``: sending failed (shown after trying to send)
-- 4: BLE (Bluetooth Low Energy)
+- 4: BLE (Bluetooth® Low Energy)
```

Comment:
hopefully such higher end tools like sphinx are happy with such special chars nowadays.

Reply:
In the preview it worked well, plus I found a discussion, that Unicode should be fine.